containers/podman-desktop
# feat: add no resources page (#1337)

### What does this PR do?

This PR displays a panel with a message when there are no resources in the resources page

@mairin i downloaded the icon from penpot. As it looks similar to a premium icon from fontawesome (https://fontawesome.com/icons/engine?f=classic&s=solid) can you confirm i can use the one I downloaded? Thank you!!

This is a first iteration and will be followed by https://github.com/containers/podman-desktop/issues/1934

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/49404737/229768079-0ef8ff9a-7236-4fbb-96f6-753027aec884.png)

### What issues does this PR fix or reference?

it is part of #1337 

### How to test this PR?

1. be sure to not have any resources
2.  go to the resources page

1b. edit the code to make the message appears when there are providers.
